fe/fe-core/src/main/java/org/apache/doris/nereids/glue/translator/ExpressionTranslator.java

```diff
@@ -258,9 +259,11 @@ public Expr visitBoundFunction(BoundFunction function, PlanTranslatorContext con
 
     @Override
     public Expr visitBinaryArithmetic(BinaryArithmetic binaryArithmetic, PlanTranslatorContext context) {
-        return new ArithmeticExpr(binaryArithmetic.getLegacyOperator(),
+        ArithmeticExpr arithmeticExpr =  new ArithmeticExpr(binaryArithmetic.getLegacyOperator(),
                 binaryArithmetic.child(0).accept(this, context),
                 binaryArithmetic.child(1).accept(this, context));
+        Utils.execWithUncheckedException(() -> arithmeticExpr.analyzeImpl(null));
```

Comment:
we have do it in `finalizeImplForNereids`, if it has any bugs, please fix it and remove this line